nitrite.go

```diff
@@ -84,7 +85,7 @@ func (h *coseHeader) AlgorithmInt() (int64, bool) {
 	return 0, false
 }
 
-type cosePayload struct {
+type CosePayload struct {
```

Comment:
I need this to be public so that I can make this structure in https://github.com/blocky/nitriding/blob/4830bbcf7c5f6e301ebdc63c1048280031a0e4f1/attester.go#L166